lib/qmp.ml

```diff
@@ -37,12 +37,18 @@ type vnc = {
   host    : string;
 }
 
+type xen_platform_pv_driver_info = {
```

Comment:
I'm using a name for the response that matches the "query-xen-platform-pv-driver-info" query command added by upstream-qemu. I think that makes it easier to understand that they are related, and perhaps avoid name conflicts in the future in case similarly-named commands are added.